### PR TITLE
Create luanderfarias.md

### PR DESCRIPTION
**Pull request by LuanderFarias**
## Describe your project in 1-3 sentences

Me and my friend @RafaeldSilva777 are building a cheap virtual reality headset with 6-dof and SteamVR support.

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
@LuanderFarias oh wow this is super cool! is there a guide that you're referencing when building this? i saw you're purchasing jumper wires, do you already have a breadboard?

also, what'd your plan for the frame? even the basic one just so it can go on your head?

also, what's your plan for hand tracking? do you need any additional sensors/modules?

and, could you please factor in shipping and taxes?

thank you so much!!

**Comment by LuanderFarias** [comment]
Thanks you, i'm following some guides of people that build a similar thing. i do have a breadboard :)

probably i'm gonna use popsticks or cardboard for the start, but i can 3d print it after i manage to decide where everything gonna go.

i worked with computer vision before, so probably gonna be easy to make a hand tracking thing (i swear). all i'm gonna need is the cameras by itself.

Oh sure sorry, editing now...

ps: sorry for the late respond

**Comment by bellesea** [comment]
> Thanks you, i'm following some guides of people that build a similar thing. i do have a breadboard :)
> 
> probably i'm gonna use popsticks or cardboard for the start, but i can 3d print it after i manage to decide where everything gonna go.
> 
> i worked with computer vision before, so probably gonna be easy to make a hand tracking thing (i swear). all i'm gonna need is the cameras by itself.
> 
> Oh sure sorry, editing now...
> 
> ps: sorry for the late respond

@LuanderFarias you're actually really quick with responding haha! sounds good, could you please link the guides that you're following? 

and, do you have access to these materials for the frame?

could you please update your file with the context you provided above (like your plan for the frame + hand tracking etc.)?

thank you sm!!

**Comment by LuanderFarias** [comment]
thanks again and sure, gonna update everything...

**Comment by LuanderFarias** [comment]
Updated :)

**Review by bellesea — APPROVED** [review]
@LuanderFarias thank you so much for pushing these new updates! can't wait to see this project come to life :)

a fabulous member of @hackclub/bank-ops will be reaching out w/ next steps shortly!

**Review by MelSmith104 — COMMENTED** [review]
hey @LuanderFarias this project sounds so cool! im dm'ing you right now so that we can chat about getting you on Bank for the grant! i can't wait to hear more about this!

File: luanderfarias.md [hunk 1]
```diff
@@ -30,12 +29,16 @@ Simplo is developed based on other open-source vr headsets out there. Here are s
...
-| 4x Arduino camera module | https://tinkersphere.com/arduino-compatible-components/944-ov7670-vga-camera-module-for-arduino.html | $44.95 |
+| 1x Arduino Micro | https://store-usa.arduino.cc/products/arduino-micro | $38.51 |
+| 1x mpu9250 | https://pt.aliexpress.com/item/1005004439490641.html?pdp_npi=2%40dis%21BRL%21R%24%2040%2C57%21R%24%2038%2C94%21%21%21%21%21%402101d1b616734837560467175eee53%2112000029185573027%21btf&_t=pvid%3A3bc2cc48-d54b-4dd2-8a53-a41f212935b8&afTraceInfo=1005004439490641__pc__pcBridgePPC__xxxxxx__1673483756&spm=a2g0o.ppclist.product.mainProduct&gatewayAdapt=glo2bra | $11.29 |
+| 2x Fresnel-lens | https://pt.aliexpress.com/item/33029909783.html?gatewayAdapt=glo2bra | $8.46 |
+| 1x HDMI VR Display | https://pt.aliexpress.com/item/32981354673.html?gatewayAdapt=glo2bra | $30.35 |
```
Comment (bellesea) [inline luanderfarias.md:39]: hiii @LuanderFarias it looks like the VR display you linked says it only supports computer & rpi?
Screenshot 2023-01-12 at 8 20 24 PM

Comment (LuanderFarias) [reply luanderfarias.md:39]: yeah. Whats the problem?

Comment (bellesea) [reply luanderfarias.md:39]: your project is arduino based?

**Review by bellesea — CHANGES_REQUESTED** [review]

**Comment by RafaeldSilva777** [comment]
## Describe your project in 1-3 sentences

I'm building a vr headset with luander cause we want to create a cheap headset since we live in a country where everything is way expansive.

## HackClub Slack Handle
<RAFAM0REIRA777>

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at https://hackclub.com/winter
- [x] I am a member of the Hack Club Slack (Join at https://hackclub.com/slack)
- [x] I have self-reviewed my project proposal.

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**